features/support/api_helpers.rb

```diff
@@ -12,7 +11,13 @@ def call_api(method, path, options={})
   if payload.nil?
     RestClient.send(method, url, options) {|response, request, result| response}
   else
-    RestClient.send(method, url, payload.to_json, options) {|response, request, result| response}
+    # can't send a payload as part of a DELETE request
```

Comment:
Not strictly true - `# can't send a payload as part of a DELETE request using the ruby rest client` would be more accurate.